_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,438 @@
+---
+id: fd3a94
+name: Links with identical accessible names and context serve equivalent purpose
+rule_type: atomic
+description: |
+  This rule checks that links with identical accessible names and context resolve to the same resource or equivalent resources.
+accessibility_requirements:
+  wcag20:2.4.4: # Link Purpose (In Context) (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgements:
+  authors:
+    - Anne Thyme Nørregaard
+    - Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to any set of any two or more HTML or SVG elements which
+
+- have the [semantic role][] of `link`, or a role that inherits from the `link` role; and
+- are in the same [web page (HTML)][]; and
+- are [included in an accessibility tree][included in the accessibility tree]; and
+- have [matching][] [accessible names][accessible name] that are not empty (`""`); and
+- have the same [programmatically determined link context][].
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching][] [accessible name][] and [programmatically determined link context][].
+
+## Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource). Resolving the links includes potential redirects, if the redirects happen instantly.
```

Comment:
I think this is better. That second sentence does not add to the expectation, it just clarifies it.

```suggestion
When followed, the links in each set of target elements resolve to the [same resource][] or to [equivalent resources](#equivalent-resource).

**Note**: Resolving the links includes potential redirects, if the redirects happen instantly.
```